### ISSUE TITLE
Do you have a concrete plan to release YOLOX-P6?

### ISSUE THREAD

**Issue by dongdongyee** [Megvii-BaseDetection/YOLOX#572]

**Comment by ruinmessi** [comment]
Yes, but with a very low priority, which is the same for 365object pretrain. Currently, our computing resources have more interesting tasks to do. For larger models on COCO, I suggest you consider Swin-Transformer or Scale-yolov4 at this time.

**Comment by zxy-95** [comment]
What's your current interesting tasts?
